idl/github.com/uber/cadence/shared.thrift

```diff
@@ -364,6 +364,7 @@ struct Decision {
 }
 
 struct WorkflowExecutionStartedEventAttributes {
+  01: optional i64 (js.type = "Long") version
```

Comment:
Let's add version to HistoryEvent as it needs to be on every type of event.  Similar to EventID, EventType and Timestamp.